packages/core/src/crypto/EcCompression.ts

```diff
@@ -0,0 +1,104 @@
+/**
```

Comment:
copy pasta

Reply:
> copy pasta

Not sure what you mean here. My point was that creating and maintaining a separate package doesn't make sense if it's AFJ specific anyway.

Reply:
Yeah I don't think extracting ourselves and maintaining it would be the way to go.

More just checking if there is a small library that does something similar as this is normally not handled in AFJ itself and currently I don't we have the expertise to back this implementation up.